Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -869,13 +869,21 @@ fileprivate func findCycle(
 ) rethrows -> (path: [Manifest], cycle: [Manifest])? {
     // Ordered set to hold the current traversed path.
     var path = OrderedCollections.OrderedSet<Manifest>()
+	
+	var fullyVisitedManifests = Set<Manifest>()
```

Comment:
formatting looks inconsistent for this line

Reply:
Thanks! Should be fixed now.